botx_fsm/storages/memory.py

```diff
@@ -0,0 +1,43 @@
+import pickle
+from datetime import timedelta
+from enum import Enum
+from typing import Any, Union
+
+from cachetools import TTLCache
+
+from botx_fsm import Key, unset
+from botx_fsm.markers import FSMStateMarker
+from botx_fsm.storages.base import BaseStorage, StateInStorage
+
+ONE_WEEK = timedelta(weeks=1).total_seconds()
+ONE_MB = 1024 * 1024
+
+
+class MemoryStorage(BaseStorage):
+    storage: TTLCache
+
+    def __init__(self, maxsize: int = ONE_MB, expire_time: int = ONE_WEEK) -> None:
+        self.storage = TTLCache(maxsize=maxsize, ttl=expire_time,)
+
+    async def get_state(self, key: Key) -> StateInStorage:
+        saved_value = self.storage.get(key.to_json())
+        if saved_value is None:
+            return unset
```

Comment:
Maybe useful  to use a walrus here?

Reply:
`saved_value` is used below, so the walrus does not make sense